src/pages/CloudModelPicker.js

```diff
@@ -20,30 +20,35 @@ import { fetchJson } from '../utils/RestUtils.js';
 import BaseWizardPage from './BaseWizardPage.js';
 import { ErrorMessage } from '../components/Messages.js';
 import { LoadingMask } from '../components/LoadingMask.js';
-import {
-  PickerButton, ActionButton, ItemHelpButton
-} from '../components/Buttons.js';
+import { PickerButton } from '../components/Buttons.js';
+import Dropdown from '../components/Dropdown.js';
+
+const FILTERS = [
+  {name: 'node-count', options: ['none', '30', '40']},
```

Comment:
30 and 40 do not seem like correct node counts, but it's a little unclear what these represent. If they are maximum, then I believe that mid-size goes to 200, right? Will an entry-size support 30?
Also, if we are doing any kind of math on them, then numbers are more appropriate than string values